src/openMVG/calibration/calibration.hpp

```diff
@@ -27,17 +27,17 @@ double computeReprojectionErrors(const std::vector<std::vector<cv::Point3f> >& o
 /**
  * @brief This function calibrates the camera.
  *
- * @param[in] imagePoints
- * @param[in] objectPoints
- * @param[in] imageSize
- * @param[in] aspectRatio
- * @param[in] cvCalibFlags
- * @param[in] cameraMatrix
- * @param[in] distCoeffs
- * @param[in] rvecs
- * @param[in] tvecs
- * @param[in] reprojErrs
- * @param[out] totalAvgErr
+ * @param[in] imagePoints Coordinates of the 2D points in each image of the sequence.
+ * @param[in] objectPoints Coordinates of the 3D points in each image of the sequence.
```

Comment:
Coordinates of the 3D points associated to the detected points in each image of the sequence.